lib/DevServerManagerV2.ts

```diff
@@ -64,7 +62,6 @@ class DevServerManagerV2 {
       if (serverInterface.setup) {
         await serverInterface.setup({
           components: projectNodes,
-          onUploadRequired,
```

Comment:
Right now, the only way to log the upload warning for components _without_ dev servers is to do it from the CLI. To keep things simple, I decided to have the CLI handle the logging for UI extensions as well. From what I could tell, there was no additional logging logic in the dev server - it just checks if the changed file is a config file and then calls the logging function passed from the CLI

It will probably make sense to revisit this in the future

Reply:
We should let the uie team know, but that sounds reasonable to me 👍